src/components/authors/authors.ts

```diff
@@ -4,17 +4,17 @@ import WebComponent, { createElementFromString } from '../../common/WebComponent
 
 class AuthorsComponent implements WebComponent {
   render(): Element {
-    return createElementFromString(`<main>
+    return createElementFromString(`<section>
         <div class="container">
```

Comment:
Instead of wrapping `<div>` with `<section>` just use this `<section>` with class="container". Same comes for tests.